packages/gamepads_android/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: gamepads_android
+description: Android implementation of gamepads, a Flutter plugin to handle gamepad input across multiple platforms.
+version: 0.1.1
+homepage: https://github.com/flame-engine/gamepads
+repository: https://github.com/flame-engine/gamepads/tree/main/packages/gamepads_android
+
+flutter:
+  plugin:
+    platforms:
+      android:
+        package: dev.markvideon.gamepads_android
```

Comment:
what package name do we use for the other platforms? should we unify?

Reply:
I don't think we ever had a native plugin under flame org before?

Imo we should use org.flame-engine or something like that

Reply:
org.flame-engine sounds good

Reply:
Is the dash a valid character name for Android packages? IIRC it isn't

Reply:
I believe so! But maybe better replace with an underscore?